File: ATSBackend/ATSBackend.Service/Models/CandidatoModel.cs
using System;

namespace ATSBackend.Service.Models
{
    public class CandidatoModel
    {
        public int IdCandidato { get; set; }
        public string Nome { get; set; }
        public string Profissao { get; set; }
        public bool Ativo { get; set; }
        public DateTime DataNascimento { get; set; }
        public int IdVaga { get; set; }
        public virtual CurriculoModel Curriculo { get; set; }
    }
}

File: ATSFrontend/src/app/components/candidato/candidato.model.ts
export interface Candidato {
    idCandidato?: number;
    idVaga?: number;
    ativo: boolean;
    nome: string;
    profissao: string;
    dataNascimento: string;
    curriculo: Curriculo;
}

export interface Curriculo {
    idCurriculo: number;
    idCandidato: number;
    idExperiencia: number;
    formacaoAcademica: string;
    experiencias: Experiencia[];
}

export interface Experiencia {
    idExperiencia: number;
    nomeEmpresa: string;
    dataInicio: string;
    dataFim: string;
}